prisma/schema.prisma

```diff
@@ -0,0 +1,50 @@
+// This is your Prisma schema file,
+// learn more about it in the docs: https://pris.ly/d/prisma-schema
+
+datasource db {
+  provider = "postgresql"
+  url      = env("DATABASE_URL")
+}
+
+generator client {
+  provider = "prisma-client-js"
+}
+
+model User {
+  id                 Int           @id @default(autoincrement())
+  firebaseUid        String?
+  name               String
+  email              String        @unique
+  digitaloceanApiKey String?
+  minecraftUsername  String?
+  worlds             WorldAccess[]
+  lastLogin          DateTime
+  createdAt          DateTime      @default(now())
+  updatedAt          DateTime
+  deletedAt          DateTime?
+}
+
+model World {
+  id          String        @id @default(uuid())
+  name        String
+  description String?
+  flavor      String
+  dropletId   String?
+  users       WorldAccess[]
```

Comment:
Same question as above - is this so that we can see which Users have access to a given World?

Reply:
See my other comment for further details, but yes.